src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace Dynamo.ViewModels
+{
+    public class PreferencesViewModel : ViewModelBase
+    {
+
+        private ObservableCollection<string> _languagesList;
+        private ObservableCollection<string> _fontSizeList;
+        private ObservableCollection<string> _numberFormatList;
+        private string _selectedLanguage;
+        private string _selectedFontSize;
+        private string _selectedNumberFormat;
+        private bool _runSettingsIsChecked;
+        private bool _runPreviewIsChecked;
```

Comment:
I think we usually do not prepend `_` for private members

Reply:
I removed the underscore in those private variables.